src/shipit_code_coverage/tests/conftest.py

```diff
@@ -120,3 +131,21 @@ def grcov_uncovered_function_artifact():
 def jsvm_uncovered_function_artifact():
     with generate_coverage_artifact('jsvm_uncovered_function.info') as f:
         yield f
+
+
+@pytest.fixture(scope='session')
+def mock_get_commit(MERCURIAL_COMMIT):
+    responses.add(
+        responses.GET,
+        'https://api.pub.build.mozilla.org/mapper/gecko-dev/rev/hg/{}'.format(MERCURIAL_COMMIT),
+        body='{} 0d1e55d87931fe70ec1d007e886bcd58015ff770'.format(MERCURIAL_COMMIT),
+        status=200)
+
+
+@pytest.fixture(scope='session')
+def mock_get_mercurial(GITHUB_COMMIT):
+    responses.add(
+        responses.GET,
+        'https://api.pub.build.mozilla.org/mapper/gecko-dev/rev/git/{}'.format(GITHUB_COMMIT),
+        body='876c7dd30586f9c6f9c99ef7444f2d73c7acfe7c {}'.format(GITHUB_COMMIT),
```

Comment:
The github commit should be the first in the response, see https://api.pub.build.mozilla.org/mapper/gecko-dev/rev/git/a5abf843f8fac8530aa5de6fb40e16547bb4a47a for example.